content/docs/hooks-reference.md

```diff
@@ -226,7 +226,7 @@ function reducer(state, action) {
   }
 }
 
-function Counter({initialState}) {
+function Counter() {
```

Comment:
This is necessary, look at line 216.

Reply:
This is required for line
`const [state, dispatch] = useReducer(reducer, initialState);`

Since Counter is a functional component, parameter it should accept is its **props** not the **initial state**